package.json

```diff
@@ -77,6 +80,14 @@
     "@babel/preset-react": "^7.6.3",
     "@babel/preset-typescript": "^7.6.0",
     "@babel/runtime": "^7.6.3",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/react": "^6.5.16",
```

Comment:
The last piece of the puzzle is upgrading the dependencies here: 

```
npm i @storybook/react@7.0.0-alpha.34
```

https://www.npmjs.com/package/@storybook/react?activeTab=versions